src/main/java/Duke/Task/Deadline.java

```diff
@@ -0,0 +1,24 @@
+package Duke.Task;
+
+public class Deadline extends Task{
+    private String when;
+    private String originalWhen;
+    public Deadline(String description, String when) {
+        super(description);
+        this.when = when.substring(3);
```

Comment:
Avoid the use of magic literals. Using a suitable variable name for the constant numbers (which describes the value) would help the understanding of the program.